### PR TITLE
Jormungandr: add info to /status and /coverage

### PR DESCRIPTION
**Pull request by azime**
- is_open_data added in status API
- region name, last_load_at added in metadata api
- New Json for status API :

``` json
{"status":
    {
    "status":"running",
    "parameters": 
    {
        ...
    }
    "is_connected_to_rabbitmq": true,
    "is_open_data": true
}
```

 -New Json for metadata API :

``` json
regions": [
    {
        "start_production_date": "20131007",
        "status": "running",
        "shape": "POLYGON((-2.558791 46.266838,-2.558791 48.584672,0.979875 48.584672,0.979875 46.266838,-2.558791 46.266838))",
        "id": "fr-pdl",
        "end_production_date": "20131215",
        "name": "Pays De La Loire",
        "last_load_at": "20140520T160025.494228"
    }
]
```

fixes http://jira.canaltp.fr/browse/NAVP-36

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Status.py [hunk 1]
```diff
@@ -42,5 +44,12 @@ class Status(Resource):
     def get(self, region):
         response = protobuf_to_dict(i_manager.dispatch({}, "status", instance_name=region), use_enum_labels=True)
         instance = i_manager.instances[region]
+        is_open_data = False
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Status.py]: if there is no db all instances are open data no ?

File: source/jormungandr/tests/coverage_tests.py [hunk 2]
```diff
@@ -46,6 +46,9 @@ def test_null_status_is_hidden(self):
         assert 'regions' in response
         assert len(response['regions']) == 1
         assert response['regions'][0]['id'] == 'main_routing_test'
+        assert 'last_load_at' in response['regions'][0]
```
Comment (antoine-de) [inline source/jormungandr/tests/coverage_tests.py:49]: you can also test that this is a valid date (I think there is a helper for that)

Comment (azime) [inline source/jormungandr/tests/coverage_tests.py:49]: OK

File: source/jormungandr/jormungandr/interfaces/v1/Status.py (hunk 1 above)
Comment (azime) [inline source/jormungandr/jormungandr/interfaces/v1/Status.py]: OK

**Comment by azime** [comment]
corrections

File: source/jormungandr/tests/coverage_tests.py (hunk 2 above)
Comment (antoine-de) [inline source/jormungandr/tests/coverage_tests.py:49]: can you also test the is_open_data boolean ?

Comment (antoine-de) [inline source/jormungandr/tests/coverage_tests.py:49]: if the sort of the region has been asked, can you test it ? you might want to use a fixture with several region loaded (this one will hide the empty region :smile: )

**Comment by azime** [comment]
corrections

File: source/jormungandr/tests/coverage_tests.py (hunk 2 above)
Comment (azime) [inline source/jormungandr/tests/coverage_tests.py:49]: ok

**Pull request merged**